src/main/java/org/jenkinsci/plugins/authorizeproject/AuthorizeProjectStrategy.java

```diff
@@ -83,11 +83,50 @@ public Authentication authenticate(Job<?, ?> project, Queue.Item item) {
     }
     
     /**
+     * Old {@link AbstractProject} based version of {@link #authenticate(Job, Queue.Item)}.
+     *
+     * @param project the project to run.
+     * @param item the item in queue, which will be a build.
+     * @return {@code true} if authentication was successful
      * @deprecated use {@link #authenticate(hudson.model.Job, Queue.Item)} instead.
      */
     @Deprecated
     public Authentication authenticate(AbstractProject<?, ?> project, Queue.Item item) {
         return authenticate((Job<?,?>)project, item);
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public AuthorizeProjectStrategyDescriptor getDescriptor() {
+        return (AuthorizeProjectStrategyDescriptor)super.getDescriptor();
+    }

```

Comment:
(Note for me)
No longer supports `Descriptor<AuthorizeProjectStrategy>`.
(1.0.0 didn't have AuthorizeProjectStrategyDescriptor)

It's OK as ownership-plugin, which is the only extension of authorize-project uses `AuthorizeProjectStrategyDescriptor`.